src/zestreleaser/towncrier/__init__.py

```diff
@@ -56,6 +56,79 @@ def _is_towncrier_wanted():
     return True
 
 
+def _report_newsfragments_sanity():
+    """Report on the sanity of the newsfragments.
+
+    I hope this is not too specific to the pyproject.toml config
+    that I am used to.
+    """
+    with open(TOWNCRIER_CONFIG_FILE, 'r') as conffile:
+        full_config = toml.load(conffile)
+    config = full_config['tool']['towncrier']
+    if 'type' in config:
+        types = [entry.get('directory') for entry in config['type']]
+    else:
+        # towncrier._settings._default_types seems too private to depend on,
+        # so hardcopy it.
+        types = ['feature', 'bugfix', 'doc', 'removal', 'misc']
+    # Where are the snippets stored?
+    directory = config.get('directory')
+    if not directory:
+        # Look for "newsfragments" directory.
+        # We could look in the config file, but I wonder if that may change,
+        # so simply look for a certain directory name.
+        fragment_directory = 'newsfragments'
+        for dirpath, dirnames, filenames in os.walk('.'):
+            if dirpath.startswith(os.path.join('.', '.')):
+                # for example ./.git
+                continue
+            if fragment_directory in dirnames:
+                directory = os.path.join(dirpath, fragment_directory)
+                break
+        if not directory:
+            # Either towncrier won't work, or our logic is off.
+            print(
+                'WARNING: could not find newsfragments directory '
+                'for towncrier.')
+            return
+    problems = []
+    correct = []
+    for filename in os.listdir(directory):
+        if filename.startswith('.'):
+            continue
+        ext = os.path.splitext(filename)[-1]
```

Comment:
With the last release of towncrier, that is no longer the case, there can be ``123.feature`` and ``123.feature.1`` as well

Reply:
Ah, you are right.